src/components/slider/slider.tsx

```diff
@@ -1200,7 +1200,7 @@ export class Slider
     const percent = (x - left) / width;
     const mirror = this.shouldMirror();
     const clampedValue = this.clamp(this.min + range * (mirror ? 1 - percent : percent));
-    let value = Number(clampedValue.toFixed(decimalPlaces(this.step)));
+    let value = Number(clampedValue.toFixed(decimalPlaces(`${this.step}`)));
```

Comment:
If this fixes precision on the slider, can you update the PR title to reflect this? If it's to fix types, can you submit this in a separate PR? If it's the latter, we can merge the separate PR first and we can skip rolling it back here.

Reply:
The parameter needs to be a string so the decimalPlaces function can also be used with BigDecimal. Previously `step` was being converted to a string in the decimalPlaces function itself, but we would lose precision if converting BigDecimal to `number` before passing it to the function. So it doesn't fix anything in slider, it's just a small, required refactor.

Reply:
Nevermind, Input isn't using decimalPlaces anymore, removing!